kubernetes/config/kube_config.py

```diff
@@ -22,151 +22,243 @@
 from kubernetes.client import configuration
 from oauth2client.client import GoogleCredentials
 
-_temp_files = []
+from .incluster_config import ConfigException
```

Comment:
Why isn't this imported from ``.config_exception``?  Consider avoiding relying on another module's imports.